src/App.tsx

```diff
@@ -589,26 +731,37 @@ function App() {
-          <button onClick={runQuery} disabled={!(error === null)} className='bg-blue-500 hover:bg-blue-700 disabled:bg-blue-300 text-white text-xl font-semibold py-2 px-4 mt-3.5 rounded mr-3 w-40' type='submit'>Run Query</button>
+          <button onClick={runQuery} disabled={!(error === null) || query === undefined} className='bg-blue-500 hover:bg-blue-700 disabled:bg-blue-300 disabled:opacity-50 text-white text-xl font-semibold py-2 px-4 mt-3.5 rounded mr-3 w-40' type='submit'>Run Query</button>
           <button onClick={() => {
-          setQuery(format(query, {
-            language: 'sqlite',
-            tabWidth: 2,
-            useTabs: false,
-            keywordCase: 'upper',
-            dataTypeCase: 'upper',
-            functionCase: 'upper',
-        }))}} disabled={!(error === null)} className='bg-blue-500 hover:bg-blue-700 disabled:bg-blue-300 text-white text-xl font-semibold py-2 px-4 mt-3.5 rounded mr-3 w-40' type='submit'>Format Code</button>
+            setQuery(format(query === undefined ? '-- Select a question to get started!.' : query, {
+              language: 'sqlite',
+              tabWidth: 2,
+              useTabs: false,
+              keywordCase: 'upper',
+              dataTypeCase: 'upper',
+              functionCase: 'upper',
+          }))}} disabled={!(error === null) || query === undefined} className='bg-blue-500 hover:bg-blue-700 disabled:bg-blue-300 disabled:opacity-50 text-white text-xl font-semibold py-2 px-4 mt-3.5 rounded mr-3 w-40' type='submit'>
+            Format Code
+          </button>
           <button onClick={() => {
-            setQuery(localStorage.getItem(`correctQuestionId-${question.id}`) || DEFAULT_QUERY);
-          }} 
-            disabled={!correctQueryMismatch}
-          className='bg-yellow-500 hover:bg-yellow-700 disabled:bg-yellow-400 disabled:opacity-50 text-white text-xl font-semibold py-2 px-4 mt-3.5 rounded mr-3 w-40' type='submit'>Load Saved</button>
+              if (!question) {
+                return;
+              }
+              setQuery(localStorage.getItem(`correctQuestionId-${question.id}`) || DEFAULT_QUERY);
+            }} 
+            disabled={!correctQueryMismatch || !question}
+            className='bg-yellow-500 hover:bg-yellow-700 disabled:bg-yellow-400 disabled:opacity-50 text-white text-xl font-semibold py-2 px-4 mt-3.5 rounded mr-3 w-40' type='submit'>
+              Load Saved
+          </button>
+          {/* Might be removed fully, servers no purpose as it is right now */}
+          {/*
           <button onClick={() => {
             if (window.confirm('Are you sure you want to reset the database?\n\nNote: This will not reset your written answers.')) {
               initDb();
             }
             }} className='bg-red-500 hover:bg-red-700 text-white text-xl font-semibold py-2 px-4 mt-4 rounded mr-3 w-40' type='submit'>Reset DB</button>
+          */}
```

Comment:
Any got input? since all it does is reload the underlying DB (which also will be done by just refreshing the page) this serves no purpose. Although a nuke save/reset state could be added?